client/components/date-range/index.js

```diff
@@ -564,7 +564,13 @@ export class DateRange extends Component {
 						},
 					} ) }
 				{ startDate && endDate && (
-					<Button className="date-range__info-btn" borderless compact onClick={ this.resetDates }>
+					<Button
+						className="date-range__info-btn"
+						borderless
+						compact
+						onClick={ this.resetDates }
+						aria-label="Reset selected dates"
```

Comment:
Should we localize this string as well?

Reply:
👍🏻

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/63879/commits/9456a33036915934b4f62bf0086b2b4d372ce8c0.